### ISSUE TITLE
[Beta] Applied Accessibility - Tests fails to check for closing </figure> tag

### ISSUE THREAD

**Issue by Asjas** [freeCodeCamp/freeCodeCamp#16420]
#### Challenge Name

https://beta.freecodecamp.org/en/challenges/applied-accessibility/improve-chart-accessibility-with-the-figure-element

#### Issue Description

I've found two problems in this challenge.

Firstly, it states no `div` tags may be used in the code. But if I leave in a closing `</div>` tag the test still passes. It's not checking for closing `div` tags.

Second, the test isn't checking if the closing `</figure>` tag is being used. It seems to be only checking for the opening tag and then passes.

#### Browser Information

* Browser Name, Version: Chrome, Version 63.0.3239.108 (Official Build) (64-bit)
* Operating System: Windows 10 Pro
* Mobile, Desktop, or Tablet: Desktop

#### Your Code

```js
<body>
  <header>
    <h1>Training</h1>
    <nav>
      <ul>
        <li><a href="#stealth">Stealth &amp; Agility</a></li>
        <li><a href="#combat">Combat</a></li>
        <li><a href="#weapons">Weapons</a></li>
      </ul>
    </nav>
  </header>
  <main>
    <section>
      
      <!-- Add your code below this line -->
      <figure>
        <!-- Stacked bar chart will go here -->
        <br>
        <figcaption>Breakdown per week of time to spend training in stealth, combat, and weapons.</figcaption>
        </div>
      <!-- Add your code above this line -->
      
    </section>
    <section id="stealth">
      <h2>Stealth &amp; Agility Training</h2>
      <article><h3>Climb foliage quickly using a minimum spanning tree approach</h3></article>
      <article><h3>No training is NP-complete without parkour</h3></article>
    </section>
    <section id="combat">
      <h2>Combat Training</h2>
      <article><h3>Dispatch multiple enemies with multithreaded tactics</h3></article>
      <article><h3>Goodbye world: 5 proven ways to knock out an opponent</h3></article>
    </section>
    <section id="weapons">
      <h2>Weapons Training</h2>
      <article><h3>Swords: the best tool to literally divide and conquer</h3></article>
      <article><h3>Breadth-first or depth-first in multi-weapon training?</h3></article>
    </section>
  </main>
  <footer>&copy; 2016 Camper Cat</footer>
</body>
```

#### Screenshot

![figure](https://user-images.githubusercontent.com/3828967/34638567-d87d1b86-f2d6-11e7-9028-781201c07e5f.jpg)

**Comment by sagarjhaa** [comment]
@raisedadead I would like to work on it.

**Comment by raisedadead** [comment]
Yes, sure go ahead, any issue that is tagged help wanted, and does not have a open PR is open for contribution. You do not need confirmation.

Goodluck and happy contributing!

**Comment by sagarjhaa** [comment]
Working on it.
